cardano-cli/src/Cardano/CLI/Types/Errors/TxCmdError.hs

```diff
@@ -217,6 +221,10 @@ renderTxCmdError = \case
     prettyError e
   TxCmdFeeEstimationError e ->
     prettyError e
+  TxCmdPoolMetadataHashError e ->
+    "Hash of the pool metadata hash is not valid:" <+> prettyException e
+  TxCmdHashCheckError url e ->
+    "Hash of the file is not valid. Url:" <+> pretty (L.urlToText url) <+> prettyException e
```

Comment:
`Pretty L.Url` could be added to `cardano-api`.

Reply:
Addressed in rebase

Reply:
[Ticket](https://input-output-rnd.slack.com/lists/T0N639Z4N/F078RTQGB1T?record_id=Rec07U4MFSTU6)